components/privacy-paragraph.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <p class="legal">
+    <em>Your email address will be used in accordance our <a
```

Comment:
The grammar seems a bit strange here. It seems like it should be 'in accordance with our' instead of `in accordance our'.